velox/functions/lib/tests/SliceTest.cpp

```diff
@@ -34,96 +35,129 @@ class SliceTest : public FunctionBaseTest {
     assertEqualVectors(expectedArrayVector, result);
     EXPECT_NO_THROW(expectedArrayVector->checkRanges());
   }
+
+  void commonTestCases() {
```

Comment:
How does this work? Do we register 2 versions of the 'slice' function somehow? Where do we do that?

Reply:
Oh, this test is originally written in `velox/functions/spark`, and after moving to `velox/functions/lib`, it won't work as we indented, though the start/length type is int when passing to the function, but it will be cast to `bigint` when parsing the expression in `Expressions.createWithImplicitCast`.

Fix this by registering a `spark_slice` function